torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -153,7 +153,7 @@ def resize_bounding_box(
     old_height, old_width = image_size
     new_height, new_width = _compute_resized_output_size(image_size, size=size, max_size=max_size)
     ratios = torch.tensor((new_width / old_width, new_height / old_height), device=bounding_box.device)
-    return bounding_box.view(-1, 2, 2).mul(ratios).view(bounding_box.shape)
+    return bounding_box.view(-1, 2, 2).mul(ratios).to(bounding_box.dtype).view(bounding_box.shape)
```

Comment:
Optional Nit: For those of us with OCDs perhaps always do last? 😄 
```suggestion
    return bounding_box.view(-1, 2, 2).mul(ratios).view(bounding_box.shape).to(bounding_box.dtype)
```

Reply:
[Ackchyually](https://knowyourmeme.com/memes/ackchyually-actually-guy) this is consistent. The dtype conversion always happens before the last `view`:

- `resize_bounding_box` as well as the format conversion functions have no final `.view()` call
- `{affine, perspective, elastic}_bounding_box` have it on the line below the change

I thought about that when I was writing the patch :grin: